CTFd/challenges.py

```diff
@@ -34,15 +35,15 @@ def listing():
     errors = get_errors()
 
     if Configs.challenge_visibility == ChallengeVisibilityTypes.ADMINS:
-        infos.append("Challenge Visibility is set to Admins Only")
+        infos.append(_l("Challenge Visibility is set to Admins Only"))
 
     if ctf_started() is False:
```

Comment:
@ColdHeat This condition seems impossible to trigger here, is this dead code?

Reply:
I believe this path is for admins to receive a visual cue that the CTF has not started. You should see it if you view the challenge page as an admin.